module/zfs/zil.c

```diff
@@ -337,6 +355,79 @@ zil_read_log_data(zilog_t *zilog, const lr_write_t *lr, void *wbuf)
 	return (error);
 }
 
+void
+zil_copy_kstat_values(zil_stats_t *zs)
```

Comment:
Weird function name, given it is not a generic copy but rather a 'reset' by copying from `empty_zil_stats`